client/src/components/to-do-list/Category.jsx

```diff
@@ -12,35 +12,30 @@ const useStyles = makeStyles((theme) => ({
   }
 }))
 
-function Category({tasks, addTask, deleteTask}) {
+function Category({tasks, addTask, openModal, deleteTask, editClick, editing}) {
   const classes = useStyles();
   const [totalTime, setTotalTime] = useState(0);
 
+  const {name} = tasks
+
   console.log('category: ', tasks);
 
   return (
   <Container>
     <Paper elevation={2} className={classes.paper} sx={{ display: 'inline-block' }}>
       <Container  sx={{ display: 'flex' }}>
-        <TextField
-        required
-        label='New Category'
-        variant='outlined'
-        />
-        <TextField
-        required
-        label='Allotted Time'
-        variant='outlined'
-        />
+        <div>{tasks.name}</div>
+        <div>Time Spent So Far: 0</div>
```

Comment:
See, next you'll have to have a variable for that string 0. I"m hoping you'll use hooks to get it, but if not it means you're going to pass it down? If so that'd mean 7 parameters